hasjob/views/index.py

```diff
@@ -307,6 +303,7 @@ def fetch_jobposts(request_args, request_values, is_index, board, board_jobs, gk
     if loadmore:
         query_params.update({'startdate': loadmore.isoformat() + 'Z', 'ph': pinned_hashids})
     if location:
+        data_filters['location_names'].append(location['name'])
```

Comment:
This only accepts one location name. For FilteredView, multiple locations will need to be passed in.

Reply:
Multiple locations are supported since `data_filters['location_names']` is initialized with supplied list of location names. This particular line is for the `/in/<location>` use case.